eth2deposit/intl/en/credentials.json

```diff
@@ -0,0 +1,14 @@
+{
+    "from_mnemonic": {
+        "msg_key_creation": "Creating your keys:\t\t"
```

Comment:
Is it possible for us to avoid including the tabs (`\t`) in the JSON files?

Reply:
We might be able to ignore it in Crowdin but it'd be ideal to remove these to not confuse translators.

Reply:
I will look into extraction this from the json and having the tabs encoded in separate print statements. 👍